data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -73,7 +73,7 @@ podman run -d --net host --privileged --name httpd \
 # Set CACHEURL to the default route, so we try to consume any images cached on the host
 # running the VM (dev-scripts configures a cache here), if none is found then the
 # downloader containers just skip and download from the internet location
-CACHEURL="http://$(ip r | grep default | head -n1 | awk '{print $3}')/images"
+CACHEURL="http://$(ip r | grep $PROVISIONING_NIC | head -n1 | awk '{print $3}')/images"
```

Comment:
Just to be safe in case any future additional route gets added, should we still grep default, but then filter the results by PROVISIONING_NIC?

Reply:
if we do that probably we can also remove the `head -n1` as there should only ever be one default route per interface?

Reply:
> Just to be safe in case any future additional route gets added, should we still grep default, but then filter the results by PROVISIONING_NIC?

+1 to also grepping for default

Reply:
Note to other reviewers, I'm not blocking on this, the PR in it's current form has been confirmed to work by more than one tester in baremetal environments - I'll leave it up to @rdoxenham if he'd like to update this or leave it as-is.

Reply:
@hardys thanks, I've grepped for default and removed the head.